homeassistant/components/tibber/sensor.py

```diff
@@ -410,6 +410,13 @@ async def async_update(self) -> None:
         ):
             return
 
+        # Price energy / tax
+        # current_price_info -> {'energy': 0.0737, 'tax': 0.1719, 'total': 0.2456, 'startsAt': '2024-08-06T09:00:00.000+02:00', 'level': 'NORMAL'}
+        await self._tibber_home.update_current_price_info()
+        res = self._tibber_home.current_price_info
+        self._attr_extra_state_attributes["price_energy"] = res.get("energy")
```

Comment:
These should in that case be added as additional sensors and not an attribute on existing ones.

Reply:
Thanks for the suggestion, I will implement it as sensors :)